spec/unit/tariff_synchronizer/cds_update_spec.rb

```diff
@@ -51,29 +53,27 @@
       # stub the file_path method to return a valid path of a real file.
       allow(cds_update).to receive(:file_path).and_return('spec/fixtures/cds_samples/tariff_dailyExtract_v1_20201010T235959.gzip')
       allow(cds_update).to receive(:filesize).and_return filesize
+
+      cds_importer = instance_double('CdsImporter')
+      allow(CdsImporter).to receive(:new).with(cds_update).and_return(cds_importer)
+      allow(cds_importer).to receive(:import).and_return inserted_oplog_records
     end
 
     it 'calls the CdsImporter import method' do
-      cds_importer = instance_double('CdsImporter')
-      expect(CdsImporter).to receive(:new).with(cds_update).and_return(cds_importer)
-      expect(cds_importer).to receive(:import).and_return inserted_oplog_records
       cds_update.import!
     end
 
     it 'marks the Cds update as applied' do
-      allow_any_instance_of(CdsImporter).to \
-        receive(:import).and_return inserted_oplog_records
       cds_update.import!
       expect(cds_update.reload).to be_applied
     end
 
     it 'logs an info event' do
-      tariff_synchronizer_logger_listener
-      allow_any_instance_of(CdsImporter).to \
-        receive(:import).and_return inserted_oplog_records
+      allow(Rails.logger).to receive(:info)
+
       cds_update.import!
-      expect(@logger.logged(:info).size).to eq 1
-      expect(@logger.logged(:info).last).to match(/Applied CDS update/)
+
+      expect(Rails.logger).to have_received(:info)
```

Comment:
I'd just remove info log tests. They add no value whatsoever and bog down our suite